R/run-nmtran.R

```diff
@@ -0,0 +1,330 @@
+#' Interface for running `NM-TRAN` on model objects
+#'
+#' Function to run `NM-TRAN` on a model object to validate its control stream
+#'  for correct coding before submission. The `NM-TRAN` dataset (`FDATA`) and
+#'  other `NONMEM` artifacts can be further inspected by keeping the run directory
+#'  around.
+#'
+#' @details
+#' `NM-TRAN` is a preprocessor for `NONMEM` that translates user-specified
+#' control stream data and instructions into a form executable by `NONMEM`.
+#'
+#' Note that `nmtran_presort` is run ahead of `NM-TRAN` for `NONMEM` versions
+#' `'nm74gf'`, `'nm74gf_nmfe'`, and `'nm75'`.
+#'  - `nmtran_presort` is an supplementary utility that preprocesses the control
+#'     stream to ensure it is in the correct format for `NM-TRAN`. It is particularly
+#'     relevant for handling specific data manipulations and ensuring compatibility
+#'     with the `NM-TRAN` executable.
+#'
+#' @param .mod A `bbi_nonmem_model` object.
+#' @param .bbi_args A named list specifying arguments to pass to `NM-TRAN`.
+#'   Similar to the `.bbi_args` argument defined in [submit_model()], though here
+#'   only `prdefault`, `tprdefault`, and `maxlim` flags are passed to `NM-TRAN`.
+#'   See [print_bbi_args()] for more details.
+#' @inheritParams submit_model
+#' @param .nonmem_version Character scalar for default version of NONMEM to use.
+#'   If left `NULL`, will look for the default version specified in the provided
+#'   `bbi` configuration file.
+#' @param clean Logical (`T`/`F`). If `FALSE`, don't delete the temporary folder
+#'   containing the `NM-TRAN` run, which will be stored in the current working
+#'   directory.
```

Comment:
"which will be stored in the current working directory" is no longer accurate.

Reply:
Realized this during your previous comment^. Will change